webapp/locales/de.json

```
@@ -293,12 +293,14 @@
       },
       "columns": {
         "name": "Name",
-        "slug": "Alias"
+        "slug": "Alias",
+        "unblock": "Entblocken"
       },
       "empty": "Bislang hast du niemanden blockiert.",
       "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü blockieren.",
       "block": "Nutzer blockieren",
-      "unblock": "Nutzer entblocken"
+      "unblock": "Nutzer entblocken",
+      "unblocked": " ist wieder entblockt"
```

```suggestion
      "unblocked": "ist wieder entblockt"
```

das Leerzeichen kann weg, oder?